probe/ebpf-check/check_load_ebpf.go

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+	"time"
+	"unsafe"
+
+	"github.com/iovisor/gobpf/elf"
+	"github.com/skydive-project/skydive/common"
+)
+
+// #cgo CFLAGS: -I../../probe/ebpf
+// #include "flow.h"
+import "C"
+
+// jmpTable { JMP_NETWORK_LAYER }
+var jmpTable = []string{"socket_network_layer"}
+
+func loadJumpMap(module *elf.Module) {
```

Comment:
Why not simply export the loadJumpMap function in ebpf.go and call it from here instead of duplicating the function ?

Reply:
because it was written independently ... and ebpf.go need gotags +ebpf ...